db/migrate/20170316204717_create_variant_details.rb

```diff
@@ -0,0 +1,15 @@
+class CreateVariantDetails < ActiveRecord::Migration
+  def change
+    create_table :variant_details, id: :uuid do |t|
+      t.uuid :split_id, null: false, index: true
+      t.string :variant, null: false
+      t.string :display_name
+      t.text :description
```

Comment:
Should these be null: false and required through validations? We don't really get any value out of having one of these rows without these two columns populated. And we're creating these lazily, so i think it makes sense to be non nullable.